### DISCUSSION TITLE
Sqlite

### DISCUSSION THREAD

**Discussion by zhao-lih** [osalvador/ReplicaDB#63] [General]
Hi Oscar, is that any work done on sqlite as sink target? If not, can you give a quick guideline on how to implement?

**Comment by osalvador** [comment]
Hi @zhao-lih ,

I am working on implementing standard JDBC connections. This will allow ReplicaDB to connect to any source and sink database using the JDBC connection, but without any vendor-specific performance improvements.

**Reply by zhao-lih** [reply]
Looking forward to it!

**Reply by osalvador** [reply]
Hi @zhao-lih 

Finally I develop a new SQLite connector in the v0.12.0 [relase](https://github.com/osalvador/ReplicaDB/releases/tag/v0.12.0)

The docs for SQLite: https://osalvador.github.io/ReplicaDB/docs/docs.html#48-sqlite-connector

Hope this help!
